test/browser/browser_topsites_section.js

```diff
@@ -116,3 +118,21 @@ test_newtab({
     await ContentTaskUtils.waitForCondition(() => (content.document.querySelector(".top-site-outer:first-child a").getAttribute("href") !== "https://bugzilla.mozilla.org"), "Topsite not removed");
   }
 });
+
+test_newtab({
+  before: setDefaultTopSites,
+  test: async function test_search_topsite_keyword() {
+    await ContentTaskUtils.waitForCondition(() => content.document.querySelector(".search-shortcut .title.pinned"), "Wait for pinned search topsites");
+
+    const searchTopSites = content.document.querySelectorAll(".title.pinned");
+    ok(searchTopSites.length >= 2, "There should be at least 2 search topsites");
+
+    searchTopSites[0].click();
+
+    return searchTopSites[0].innerText;
+  },
+  after: searchTopSiteTag => {
```

Comment:
Aha! Passing data from test content to main. 👍 

I believe this can be a plain method: `after(searchTopSiteTag) {`